services/src/test/groovy/org/jfrog/artifactory/client/GradlePackageTypeRepositoryTests.groovy

```diff
@@ -99,8 +97,10 @@ public class GradlePackageTypeRepositoryTests extends BaseRepositoryTests {
             // local
             assertThat(checksumPolicyType, CoreMatchers.nullValue())
             assertThat(handleReleases, CoreMatchers.is(expectedSettings.getHandleReleases())) // always in resp payload
-            assertThat(maxUniqueSnapshots, CoreMatchers.is(expectedSettings.getMaxUniqueSnapshots())) // always in resp payload
-            assertThat(maxUniqueSnapshots, CoreMatchers.is(expectedSettings.getMaxUniqueSnapshots()))  // always in resp payload
+            assertThat(maxUniqueSnapshots, CoreMatchers.is(expectedSettings.getMaxUniqueSnapshots()))
+            // always in resp payload
+            assertThat(maxUniqueSnapshots, CoreMatchers.is(expectedSettings.getMaxUniqueSnapshots()))
+            // always in resp payload
```

Comment:
Seems like a duplicate comment. See lines above. Let's put it once above alll relevant lines.
The same goes for the **IvyPackageTypeRepositoryTests, MavenPackageTypeRepositoryTests, P2PackageTypeRepositoryTests** and a few other classes.